api.go

```diff
@@ -926,21 +992,23 @@ func (cl *Client) GetWithdrawal(ctx context.Context, req *GetWithdrawalRequest)
 	return &res, nil
 }
 
-// ListBeneficiariesResponseRequest is the request struct for ListBeneficiariesResponse.
-type ListBeneficiariesResponseRequest struct{}
+// ListBeneficiariesRequest is the request struct for ListBeneficiaries.
+type ListBeneficiariesRequest struct {
+	BankRecipient string `json:"bank_recipient" url:"bank_recipient"`
+}
 
-// ListBeneficiariesResponseResponse is the response struct for ListBeneficiariesResponse.
-type ListBeneficiariesResponseResponse struct {
+// ListBeneficiariesResponse is the response struct for ListBeneficiaries.
+type ListBeneficiariesResponse struct {
 	Beneficiaries []beneficiary `json:"beneficiaries"`
 }
 
-// ListBeneficiariesResponse makes a call to GET /api/1/beneficiaries.
+// ListBeneficiaries makes a call to GET /api/1/beneficiaries.
 //
 // Returns a list of bank beneficiaries.
 //
 // Permissions required: <code>Perm_R_Beneficiaries</code>
-func (cl *Client) ListBeneficiariesResponse(ctx context.Context, req *ListBeneficiariesResponseRequest) (*ListBeneficiariesResponseResponse, error) {
```

Comment:
Are we confident no one would be using this? Was just considering deprecating, and adding the new function separately. But I guess as we're < 1.0.0 in terms of versioning, then breaking changes are expected

Reply:
I think we are fine. Not sure why the function was originally named `ListBeneficiariesResponse`